tests/time_tests/test_runner/conftest.py

```diff
@@ -384,6 +384,9 @@ def pytest_runtest_makereport(item, call):
     data = item._request.test_info["db_info"].copy()
     data["results"] = item._request.test_info["results"].copy()
     data["raw_results"] = item._request.test_info["raw_results"].copy()
+    name, frequency = get_cpu_info()
+    data["cpu_info"] = {"cpu_name": name,
```

Comment:
@asomsiko in performance tests "cpu_info" contains `"cpu_name" @ "cpu_frequency"` (1 field instead of 2). Do we need to handle this information in other way? I think the best solution is to align fields in all collections to simplify enabling on reporter side

Reply:
yes, values should look the same as in other tests.